frontend/package.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
we should add the correct PWA 6.1 peerDependencies here

Reply:
Updated peerDependencies for pwa-common and pwa-core to 6.1.0